webrtc-stats.html

```diff
@@ -1889,6 +1913,15 @@
                   or UDP. It is similar to the TIAS defined in [[!RFC3890]], i.e., it is measured
                   in bits per second and the bitrate is calculated over a 1 second window.
                 </p>
+                <p>
+                  Implementations that do not calculate a sender-side estimate should leave this undefined.
+                  Additionally, the value should be undefined for candidate pairs that were never used.
+                  For pairs in use, the estimate is normally no lower than the bitrate for
+                  the packets sent at <dfn>lastPacketSentTimestamp</dfn>, but might be higher.
+                  For candidate pairs that are not currently in use but were used before,
+                  implementations can use the last calculated estimate provided that no further
+                  packets are sent.
```

Comment:
Assuming the case that the pair was active and now inactive, 
1.  no packets (including consent packet) being sent/received, use last estimate, we know when that last estimate was made because of the last packet timestamp.
2. no media packets are being sent but consent packets, return undefined.

question, would doing both make sense or just the case 2.